man/systemd.xml

```diff
@@ -372,6 +372,14 @@
   <refsect1>
     <title>Signals</title>
 
+    <para>The service listens to various UNIX process signals that can be used to request various actions
+    asynchronously. The signal handling is enabled very early during boot, before any further processes are
+    invoked. However, a supervising container manager or similar that intends to request these operations via
+    this mechanism must take into consideration that this functionality is not available during the earliest
+    initialization phase. An <function>sd_notify()</function> notification message carrying the
+    <varname>X_SYSTEMD_SIGNALS_LEVEL=2</varname> field is emitted once the signal handlers are enabled, see
```

Comment:
Given the intention here is to add new levels when new signals are added, I think it would be better to immediately structure the following list in a per-level structured table or list or so. That way, when a new level is added, the manpage structure doesn't have to be changed, but just a new structured entry/list/etc added, and it's clearer for readers/users

Reply:
done